composer/core/__init__.py

```diff
@@ -45,5 +45,5 @@
     'JSON',
     'MemoryFormat',
     'TrainerMode',
-    'validate_eval_automicrobatching',
+    'validate_evaluator',
```

Comment:
Why is this included as part of composer.core? This is backwards breaking right?

Reply:
https://github.com/mosaicml/composer/pull/3100#discussion_r1518223176